docs/area-owners.json

```diff
@@ -9,6 +9,67 @@
       "label": "arch-wasm"
     }
   ],
+  "operatingSystems": [
+    {
+      "lead": "",
+      "owners": [
+        ""
+      ],
+      "label": "os-alpine"
```

Comment:
We don't have any lead owning alpine in general. Should we list the lead and the owner as symbolic names to allow creating queries for the Alpine?

Reply:
@danmoseley How do you think about the OS and architecture entries where we don't have a lead and/or owners defined? I presume those issues continue to be fully owned by the owners of the area label, whereas when there's an `os-` or `arch-` owner, that overrides the area ownership in terms of responsibility/accountability.

Reply:
I'm going to merge and we can revise the `os-` and `arch-` data if we decide there are values we should fill in.

Reply:
Right, some have owners and some don't, and that makes sense to me.

Reply:
FWIW, we can query for issues that have an area but don't have an area lead:

[`is:open is:issue no:area-lead not no:area`](https://issuesof.net/?q=is%3Aopen%20is%3Aissue%20no%3Aarea-lead%20not%20no%3Aarea)